### PR TITLE
注文リポジトリから取得する注文の情報を追加しました

### PR DESCRIPTION
**Pull request by SekiMoemi**

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 1]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
 				order.setUserId(rs.getInt("user_id"));
 				order.setStatus(rs.getInt("status"));
 				order.setTotalPrice(rs.getInt("total_price"));
-
+				order.setOrderDate(rs.getDate("order_date"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 2]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationName(rs.getString("destination_name"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 3]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationEmail(rs.getString("destination_email"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 4]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationZipcode(rs.getString("destination_zipcode"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 5]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationPref(rs.getString("destination_pref"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 6]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationMunicipalities(rs.getString("destination_municipalities"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 111).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 7]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationAddress(rs.getString("destination_address"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 8]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDestinationTel(rs.getString("destination_tel"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:70]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:70]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 9]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setDeliveryTime(rs.getTimestamp("delivery_time"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 10]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+				order.setPaymentMethod(rs.getInt("payment_method"));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:72]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:72]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 14.

File: src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java [hunk 11]
```diff
@@ -60,7 +60,17 @@ public class OrderRepository {
...
+
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/OrderRepository.java:73]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by yamada-kensyu — APPROVED** [review]
承認します

**Pull request merged**